internal/templates/element/tests.go

```diff
@@ -165,6 +165,24 @@ func Test{{toUpper .ElementName}}IsRandom(t *testing.T) {
 	}
 }
 
+func TestByte{{.ElementName}}(t *testing.T) {
+	modulus := {{.ElementName}}Modulus()
```

Comment:
add test for sample = 0, and sample = modulus << 64 . Convert sample (from big.Int to bytes) with a larger byte slice than ElementLimbs, so we test these 2 different cases (larger input than modulus, 0)